source/module_symmetry/symmetry.cpp

```diff
@@ -156,21 +156,29 @@ void Symmetry::analy_sys(const UnitCell_pseudo &ucell, std::ofstream &ofs_runnin
 	}
 
 
-	Symm_Other::print1(ibrav, cel_const, ofs_running);
-
+	//Symm_Other::print1(ibrav, cel_const, ofs_running);
+        Symm_Other::print1(real_brav, cel_const, ofs_running);
 	this->change_lattice();
     //this->pricell();         // pengfei Li 2018-05-14 
          //for( iat =0 ; iat < ucell.nat ; iat++)   
 //         std::cout << " newpos_now = " << newpos[3*iat] << " " << newpos[3*iat+1] << " " << newpos[3*iat+2] << std::endl;
-	ModuleBase::GlobalFunc::OUT(ofs_running,"ibrav",ibrav);
-    this->setgroup(this->symop, this->nop, this->ibrav);
-    //now select all symmetry operations which reproduce the lattice
-    //to find those symmetry operations which reproduce the entire crystal
-    this->getgroup(this->nrot, this->nrotk, ofs_running);
-    // find the name of point group
-    this->pointgroup(this->nrot, this->pgnumber, this->pgname, this->gmatrix, ofs_running);
-    ModuleBase::GlobalFunc::OUT(ofs_running,"POINT GROUP", this->pgname);
-    //write();
+	test_brav = true; // output the real ibrav and point group
+	ModuleBase::GlobalFunc::OUT(ofs_running,"ibrav",real_brav);
```

Comment:
it might be better if we notify the user that their input lattice type is not the optimal one